wetmap/src/components/newModals/shopModal/index.tsx

```diff
@@ -89,16 +55,25 @@ export default function ShopModal(props) {
   };
 
   const handleShopModalClose = () => {
+    props.onModalCancel();
     setSelectedShop({ ...selectedShop, id: 0, orgName: '' });
     setItineraryList('');
-    setShopModal(false);
   };
-  const fileUploaderRef = useRef<HTMLInputElement>(null);
 
   return (
     <>
     {selectedShop[0] && (
-      <ShopModalView shopModelName={selectedShop[0].orgName} shopDescription="test"/>
+      <ShopModalView 
+        setSelectedID={setSelectedID}
```

Comment:
The complaint here is that setSelectedID is not typed correctly over on the View file, you'll need to figure out what type a react set state is to make Ts happy